backend/Controller.cs

```diff
@@ -54,5 +79,69 @@ public ActionResult<IEnumerable<Leaf>> GetLeafz()
             return Program.leafs;
         }
 
+        public static void processData() 
```

Comment:
Pascal case for public members? :)
```suggestion
        public static void ProcessData() 
```